libs/feature/dataviz/src/lib/geo-table-view/geo-table-view.component.spec.ts

```diff
@@ -21,6 +21,14 @@ import XYZ from 'ol/source/XYZ'
 import { Subject } from 'rxjs'
 
 import { GeoTableViewComponent } from './geo-table-view.component'
+import { Geometry } from 'ol/geom'
+
+class ResizeObserverMock {
```

Comment:
This type of generic mock can go in the `test-setup` files in each lib, in order to avoid cluttering each separate test file